build_src.js

```diff
@@ -45,12 +47,34 @@ module.exports = function buildSrc() {
                     sourcemap: true,
                     strict: false
                 });
+            });
+        const worker = rollup
+            .rollup({
+                input: './modules/worker.js',
+                plugins: [
+                    nodeResolve({
+                        module: true,
+                        main: true,
+                        browser: false
+                    }),
+                    commonjs(),
+                    json()
+                ],
             })
-            .then(function () {
+            .then(function(bundle) {
+                return bundle.write({
+                    format: 'iife',
+                    file: 'dist/iD-worker.js',
```

Comment:
It's fine to generate a worker.js, but we really should leave `dist/iD.js` alone

Reply:
Do you mean not putting `iD-worker.js` to the `dist` folder? or renaming it `worker.js`?